src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -0,0 +1,155 @@
+    } else if (any.is<double>()) {
+        return py::cast<py::object>(PyFloat_FromDouble(any.as<double>()));
+    }
+    // Check for bool
+    else if (any.is<bool>()) {
+        return py::cast<py::object>(any.as<bool>() ? Py_True : Py_False);
+    }
+    // Check for std::vector<std::string>
+    else if (any.is<std::vector<std::string>>()) {
+        auto val = any.as<std::vector<std::string>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyObject* str_val = PyUnicode_FromString(it.c_str());
+            PyList_Append(list, str_val);
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::vector<int>
+    else if (any.is<std::vector<int>>()) {
+        auto val = any.as<std::vector<int>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyList_Append(list, PyLong_FromLong(it));
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::vector<int64_t>
+    else if (any.is<std::vector<int64_t>>()) {
+        auto val = any.as<std::vector<int64_t>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyList_Append(list, PyLong_FromLong(it));
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::vector<unsigned int>
+    else if (any.is<std::vector<unsigned int>>()) {
+        auto val = any.as<std::vector<unsigned int>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyList_Append(list, PyLong_FromLong(it));
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::vector<float>
+    else if (any.is<std::vector<float>>()) {
+        auto val = any.as<std::vector<float>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyList_Append(list, PyFloat_FromDouble((double)it));
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::vector<double>
+    else if (any.is<std::vector<double>>()) {
+        auto val = any.as<std::vector<double>>();
+        PyObject* list = PyList_New(0);
+        for (const auto& it : val) {
+            PyList_Append(list, PyFloat_FromDouble(it));
+        }
+        return py::cast<py::object>(list);
+    }
+    // Check for std::tuple<unsigned int, unsigned int>
+    else if (any.is<std::tuple<unsigned int, unsigned int>>()) {
+        auto val = any.as<std::tuple<unsigned int, unsigned int>>();
+        PyObject* tuple = PyTuple_New(2);
+        PyTuple_SetItem(tuple, 0, PyLong_FromUnsignedLong((unsigned long)std::get<0>(val)));
+        PyTuple_SetItem(tuple, 1, PyLong_FromUnsignedLong((unsigned long)std::get<1>(val)));
+        return py::cast<py::object>(tuple);
+    }
+    // Check for std::tuple<unsigned int, unsigned int, unsigned int>
+    else if (any.is<std::tuple<unsigned int, unsigned int, unsigned int>>()) {
```

Comment:
Is it covered by tests? I see only test for tuple with two elements.

Reply:
I'll cover it by another PR.